lib/Doctrine/DBAL/Platforms/MySqlPlatform.php

```diff
@@ -973,7 +973,7 @@ public function getDropIndexSQL($index, $table=null)
             return $this->getDropPrimaryKeySQL($table);
         }
 
-        return 'DROP INDEX ' . $indexName . ' ON ' . $table;
+        return 'DROP INDEX `' . $indexName . '` ON ' . $table;
```

Comment:
The quoting of an identifier (any identifier) should be performed by using `$this->quoteStringLiteral()`, and not by hardcoding backticks